Competition2017/src/competition/subsystems/climbing/commands/RopeAlignerCommand.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystems.climbing.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.climbing.RopeAlignerSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class RopeAlignerCommand extends BaseCommand {
+
+    private final RopeAlignerSubsystem ropeAlignerSubsystem;
+    
+    @Inject
+    public RopeAlignerCommand(RopeAlignerSubsystem ropeAlignerSubsystem) {
+        this.ropeAlignerSubsystem = ropeAlignerSubsystem;
+        this.requires(ropeAlignerSubsystem);
+    }
+    
+    @Override
+    public void initialize() {
+        
```

Comment:
Commands should log that they are being initialized. See other commands for examples